packages/ios_platform_images/example/ios/RunnerTests/UIImage+ios_platform_imagesTests.m

```diff
@@ -0,0 +1,34 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import ios_platform_images;
+@import XCTest;
+
+// The tests test the UIImage extension which is a public API intended for use from native code
+// outside of the plugin (see package README). Any change that requires changing existing tests
+// in this file (unless it's just to reflect changes to the test assets) is a BREAKING CHANGE for
+// the package.
+@interface UIImageExtensionTests : XCTestCase
+@end
+
+@implementation UIImageExtensionTests
+
+- (void)testMultiResolutionImageUsesBest {
+  UIImage *image = [UIImage flutterImageWithName:@"assets/multisize.png"];
+  XCTAssertNotNil(image);
+  // The 1x image height is is 125px, and the 2x is 250px.
+  XCTAssertEqualWithAccuracy(image.size.height, 250, 0.00001);
```

Comment:
if this is testing the 1x "assets/multisize.png" shouldn't it be 125? Seems to be passing though so 🤷

Reply:
This is testing that we are getting `assets/2.0x/multisize.png` instead of `assets/multisize.png` which is the documented behavior (as long as the screen scale is >1x which is true for anything we are going to test on).

Reply:
is this something we can add asserts for, or some note about the screen size?

Reply:
Sure, that makes sense. I can make it pass on a 1x screen even though that's not likely to actually come up in practice, with some comments, so the test is clearer. Good thought!

Reply:
This should be much clearer, and will now pass even if run on a 1x screen.